micrometer-core/src/main/java/io/micrometer/core/instrument/binder/kafka/KafkaConsumerMetrics.java

```diff
@@ -241,15 +250,15 @@ private void registerNotificationListener(String type, BiConsumer<ObjectName, Ta
         NotificationListener notificationListener = (notification, handback) -> {
             MBeanServerNotification mbs = (MBeanServerNotification) notification;
             ObjectName o = mbs.getMBeanName();
+            logger.trace("Got a JMX notification for object: {} with properties: {}", o.getCanonicalName(), o.getKeyPropertyListString());
```

Comment:
We might want to guard this with a `isTraceEnabled` conditional. It looks like `getKeyPropertyListString` may not be the cheapest call to make, with the following comment in its method body:

```
// BEWARE : we rebuild the propertyliststring at each call !!
```

Reply:
Removed the logging.

Reply:
:) Funny, I just wanted to do the same. Thanks.